PR#1-Added hover feature for Social Icons

Jawad-Nazari opened the pull request:
In this PR I did the following:

- Added the `index.html` file.

- Added the `style.css` file.

- Added the linters file and ensured there was no linter error

- Documented the project in a professional way.

- And in the last commit added hover style to the social icons

levy002 reviewed (commented):
## Hello @Jawad-Nazari 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: :red_circle: 

You have submitted a project, that doesn't use the professional README. Kindly make sure to use the[ Microverse README template](https://github.com/microverseinc/readme-template) and make sure to customize it according to your project. Kindly fix that and submit it again. :wink: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Jawad-Nazari commented:
Hello @levy002 
I hope your doing well.
Thanks for the time and reviewing the README.md file. 
I have updated the README file according to Microverse template and added more explanation to it and then did commit it into the feature.branch and also summited a new Code Review Request.
I would appreciate if you review it again.

aamirkhan2478 reviewed (commented):
Hi @Jawad-Nazari , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that all changes should be made on the feature branch and not the default branch, this way, CRs can use GitHub features(inline comments) for proper review.
So kindly fix this also.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
